estate/models/estate.py

```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "This model describes the attributes of a property in real estate"
```

Comment:
In vauxoo we usually use the model's name in the description. I'll explain why this is later.

```suggestion
    _description = "Estate Property"
```